spot_rl_experiments/spot_rl/launch/local_only.sh

```diff
@@ -3,13 +3,15 @@ tmux kill-session -t roscore
 tmux kill-session -t img_pub
 tmux kill-session -t propio_pub
 tmux kill-session -t tts_sub
+tmux kill-session -t spotWorld_static_tf2_pub
 sleep 1
 echo "Starting roscore tmux..."
 tmux new -s roscore -d '$CONDA_PREFIX/bin/roscore'
 echo "Starting other tmux nodes.."
 tmux new -s img_pub -d '$CONDA_PREFIX/bin/python -m spot_rl.utils.img_publishers --local'
 tmux new -s propio_pub -d '$CONDA_PREFIX/bin/python -m spot_rl.utils.helper_nodes --proprioception'
 tmux new -s tts_sub -d '$CONDA_PREFIX/bin/python -m spot_rl.utils.helper_nodes --text-to-speech'
+tmux new -s spotWorld_static_tf2_pub -d 'rosrun tf2_ros static_transform_publisher 0 0 0 0 0 0 /map /spotWorld'
```

Comment:
do we need this to launch the script if we do not use Aria?

Reply:
Yes, because now you can also visualize real-time location of spot inside rviz. Without needing aria.

For visualizing it in rviz, you will need this publisher.

Reply:
@jimmytyyang Does the above make sense to you?